judgments/feeds.py

```diff
@@ -139,3 +115,90 @@ def feed_extra_kwargs(self, obj):
         extra_kwargs["total"] = int(obj["search_response"].total)
         extra_kwargs["page"] = obj["page"]
         return extra_kwargs
+
+
+class LatestJudgmentsFeed(JudgmentsFeed):
+    def link(self, obj):
+        page = obj.get("page", 1)
+        return f"/{obj['slug']}/atom.xml/?page={page}&order={obj.get('order')}"
```

Comment:
One errant slash (I think):

```suggestion
        return f"/{obj['slug']}/atom.xml?page={page}&order={obj.get('order')}"
```